src/frontend/admin/package.json

```diff
@@ -18,15 +18,27 @@
     "@emotion/server": "11.11.0",
     "@emotion/styled": "11.11.0",
     "@emotion/cache": "11.11.0",
+    "@faker-js/faker": "7.6.0",
     "@fontsource/roboto": "5.0.1",
+    "@hookform/resolvers": "2.9.11",
+    "@jest/globals": "29.5.0",
     "@mui/icons-material": "5.11.16",
+    "@mui/lab": "5.0.0-alpha.124",
     "@mui/material": "5.13.2",
+    "@mui/x-data-grid": "5.17.25",
+    "@mui/x-date-pickers": "6.0.1",
     "@next/font": "13.4.3",
```

Comment:
`@next/font` is useless now
https://nextjs.org/docs/messages/built-in-next-font